atom/browser/atom_browser_main_parts.cc

```diff
@@ -236,7 +239,13 @@ void AtomBrowserMainParts::PreMainMessageLoopRun() {
       AtomWebUIControllerFactory::GetInstance());
 #endif  // BUILDFLAG(ENABLE_PDF_VIEWER)
 
-  brightray::BrowserMainParts::PreMainMessageLoopRun();
+  content::WebUIControllerFactory::RegisterFactory(
+      WebUIControllerFactory::GetInstance());
```

Comment:
Lets merge the contents of `WebUIControllerFactory` into `AtomWebUIControllerFactory` and eliminate it.